### ISSUE TITLE
Swipe to Edit/Delete lacks discoverability

### ISSUE THREAD

**Issue by autoreleasefool** [autoreleasefool/approach#282]
Have already have 1 request for someone looking for this functionality (they ended up finding it later on their own)
